sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -323,7 +323,7 @@ class SparkSession private(
    *  // |-- age: integer (nullable = true)
    *
    *  dataFrame.createOrReplaceTempView("people")
-   *  sparkSession.sql("select name from people").collect.foreach(println)
+   *  sparkSession.sql("select name from people").show
```

Comment:
I imagine this is an OK modification but it's not really a typo fix. I'd avoid changes that aren't fixing problems